apps/server/src/schema/feedback/feedback-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { type Feedback } from '@repo/database';
+import { logger } from '@repo/logger';
+import { Environment, MODE } from '@repo/mode';
+import { AError } from '@repo/utils';
+
+export const postFeedbackToSlack = async (feedback: Feedback): Promise<AError | undefined> => {
+  const slackMessage = {
+    text: `New ${MODE !== Environment.Production ? 'Test ' : ''} Feedback Received: \n*Type:* ${feedback.type} \n*Message:* ${feedback.message}`,
+  };
+
+  const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL_PUBLIC_FEEDBACK;
+
+  if (!slackWebhookUrl) {
+    return new AError('No slack webhook URL provided');
+  }
```

Comment:
```suggestion
  if (!slackWebhookUrl) return;
```
I think it is fine if we just return. This way devs without the URL will be able to test feedback without flooding the channel.